### DISCUSSION TITLE
How to create a new release to the VS Code Marketplace

### DISCUSSION THREAD

**Discussion by b-ned** [AltimateAI/vscode-dbt-power-user#480] [General]
Hi @mdesmet,

What is the process for creating a new release to the VS Code Marketplace?
In the past months I've added a couple of new features and I would love to see them in a new release,

Is this something we have to do manually after a PR is merged?

Kind regards,

Bas

**Comment by mdesmet** [comment]
Releasing can only be done by maintainers. I will do a release shortly.

**Reply by b-ned** [reply]
Thanks!
